src/GitHub.Api/Resources/.gitignore

```
@@ -1,13 +1,16 @@
-/[Ll]ibrary/
-/[Tt]emp/
-/[Oo]bj/
-/[Bb]uild/
-/[Bb]uilds/
+[Ll]ibrary/
+[Tt]emp/
+[Oo]bj/
+[Bb]uild/
+[Bb]uilds/
+[Ll]ogs/
+Assets/AssetStoreTools*
```

Anyone know why this wildcard `*` is here? I only see one folder for that asset.
```suggestion
Assets/AssetStoreTools/
```

---

Right. Forgot about the meta file, thanks.

---

You know what, you have a point, maybe it is better not to have this ignore here, it's going to be confusing for people if they want to commit the tooling into their repo and it's ignored by default.